parsers/CY.py

```diff
@@ -16,6 +16,18 @@ class CyprusParser:
         'Φωτοβολταϊκή Εγκατεστημένη Ισχύς': 'solar',
         'Εγκατεστημένη Ισχύς Βιομάζας': 'biomass'
     }
+    STATIC_CAPACITIES = {
```

Comment:
I'm not sure if `STATIC_CAPACITIES` are necessary, and if they are it might be better to default to capacity values in `zones.json` as opposed to from the parser.

@FelixDQ do you know what static values are used as fallbacks when capacities are pulled dynamically from a parser?

Reply:
@gwpicard they currently aren't https://www.electricitymap.org/zone/CY. Alternatively I could only set the ones that are zero.

Reply:
Hello,
In theory the static capacities from zones.json should be automatically used as a fallback (so they are not needed in the parser).
If this fallback mechanism doesn't work, then I suggest opening an issue so we can look into it!

Reply:
@corradio If that is the case, then there shouldn't be a need to have `STATIC_CAPACITIES` included in the parser. 

From what I'm seeing at [https://www.electricitymap.org/zone/CY](https://www.electricitymap.org/zone/CY), the values for biomass, wind and solar are the same as what is specified in `zones.json` even though they aren't currently specified in the parser. The '?' are because we don't currently have any capacity values for the other production modes.

So it looks as though the `zones.json` values are used as a fallback unless I'm missing something @veqtrus?

Reply:
The values for biomass, wind, solar, and oil don't have '?' as they are returned by the parser. What the parser doesn't currently return is the zero capacities, as the parser looks for which capacities are mentioned in the source. Therefore, no, the backend does not correctly fallback to zones.json, as that file has zero values for other production types.

Reply:
I see. Perhaps the best way to do it so that it is simpler would be to have a way for the backend to fall back to values in `zones.json` even in the case where the parser only returns some capacity values.

Essentially, we only use dynamic capacities if we are able to return all of them from the parser, and if not, we fall back to `zones.json`. How does that sound? :)

Reply:
@gwpicard I may have understood you wrong, but if you fall back to zones.json if some values are missing from the parser output, the parser will never be used as it doesn't return the zero capacities. What should happen is that for those values that are missing from the parser output, values returned recently or the ones from zones.json should be used.

Reply:
>What should happen is that for those values that are missing from the parser output, values returned recently or the ones from zones.json should be used

Yes, this is also an acceptable approach! In essence, I think whatever would avoid having capacities stored in the parser (like `STATIC_CAPACITIES`) is preferable. And if we can do that using your above approach that would be great.